js/scripts.js

```diff
@@ -0,0 +1,118 @@
+var newGameElem = document.getElementById('js-newGameElement');
+var pickElem = document.getElementById('js-playerPickElement');
+var resultsElem = document.getElementById('js-resultsTableElement');
+var playerPointsElem = document.getElementById('js-playerPoints');
+var playerNameElem = document.getElementById('js-playerName');
+var computerPointsElem = document.getElementById('js-computerPoints');
+var playerPickElem = document.getElementById('js-playerPick');
+var computerPickElem = document.getElementById('js-computerPick');
+var playerResultElem = document.getElementById('js-playerResult');
+var computerResultElem = document.getElementById('js-computerResult');
+
+var gameState = 'notStarted';	//started, ended
+var player = {
+	name: '',
+	score: 0
+};
+var computer = {
+	score: 0
+};
+
+// Events
+newGameBtn.addEventListener('click', newGame);
+pickRock.addEventListener('click', function() { playerPick('kamień') });
+pickPaper.addEventListener('click', function() { playerPick('papier') });
+pickScissors.addEventListener('click', function() { playerPick('nożyce') });
+
+// Functions
+function setGameElements() {
+	switch(gameState) {
+		case 'started':
+			newGameElem.style.display = 'none';
+			pickElem.style.display = 'block';
+			resultsElem.style.display = 'block';
+		break;
+		case 'ended':
+			newGameBtn.innerText = 'Jeszcze raz';
+		case 'notStarted':
+		default:
+			newGameElem.style.display = 'block';
+			pickElem.style.display = 'none';
+			resultsElem.style.display = 'none';
+	}
+}
+
+function newGame() {
+	player.name = prompt('Graczu, podaj swoje imię', 'Twoje imię');
+	if (player.name) {
+		player.score = computer.score = 0;
+		gameState = 'started';
+		setGameElements();
+
+		playerNameElem.innerHTML = player.name;
+		setGamePoints();
+	}
+}
+
+function getComputerPick() {
+	var possiblePicks = ['kamień', 'papier', 'nożyce'];
+	return possiblePicks[Math.floor(Math.random()*3)];
+}
+
+function checkRoundWinner(playerPick, computerPick) {
+	playerResultElem.innerHTML = computerResultElem.innerHTML = '';
+
+	var winnerIs = 'player';
+
+	if (playerPick == computerPick) {
+		winnerIs = 'noone';
+	} else if (
+			(computerPick == 'kamień' && playerPick == 'nożyce') ||
+			(computerPick == 'nożyce' && playerPick == 'papier') ||
+			(computerPick == 'papier' && playerPick == 'kamień')) {
+		winnerIs = 'computer';
+	}
+
+	if (winnerIs == 'player') {
+		playerResultElem.innerHTML = 'Wygrana!';
+		player.score++;
+		playerPointsElem.innerHTML = player.score;
+	} else if (winnerIs == 'computer') {
+		computerResultElem.innerHTML = 'Wygrana!';
+		computer.score++;
+		computerPointsElem.innerHTML = computer.score;
+	}
+}
+
+function setGamePoints() {
+	playerPointsElem.innerHTML = player.score;
+	computerPointsElem.innerHTML = computer.score;
+}
+
+function endGame() {
+	if (player.score == 10) {
+		alert(player.name + ', zdobyłeś ' + player.score + ' punktów. Wygrałeś.');
+		gameState = 'ended';
+	} else if (computer.score == 10) {
```

Comment:
bezpieczniej jest porównywać przez `===` - to tak na przyszłość. Jak porównujesz przez `==` to upewnij się, że z pełną świadomością